index.js

```diff
@@ -965,6 +965,27 @@ class Encore {
         return this;
     }
 
+    /**
+     * Configure the style-loader.
+     *
+     * https://github.com/webpack-contrib/style-loader#Options
+     *
```

Comment:
Can we add:

The style-loader is used only if you also call `Encore. disableCssExtraction()`.

Reply:
I've done that.